Ibnul Ekram

```diff
@@ -0,0 +1,20 @@
+import junit.framework.Assert;
+
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+public class NFLTest
+
+{
+@Test
+public void TestTitle()
+
+{
+WebDriver dr= new FirefoxDriver();
+dr.get("http://scores.espn.go.com/nfl/scoreboard");
```

Comment:
Can you update this test to start on espn.go.com and click on the link for the NFL scoreboard?